public/js/listener.js

```diff
@@ -3,7 +3,21 @@
 // pass 'latest headlines' request on to request module.
 var Listener = (function() {
 
-
+  var latestButton = document.getElementById('latestButton');
+
+  latestButton.addEventListener('click', function(callback) {
```

Comment:
(callback) as an argument in an eventListener's handler will hold the value of the event object being passed into the event handler